pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -76,10 +76,19 @@ def __init__(self, *args, **kargs):
             fillOutline  (bool) If True, an outline surrounding the *fillLevel* area is drawn.
             fillBrush    Fill to use when fillLevel is specified.
                          May be any single argument accepted by :func:`mkBrush() <pyqtgraph.mkBrush>`
-            stepMode     If True, two orthogonal lines are drawn for each sample
-                         as steps. This is commonly used when drawing histograms.
-                         Note that in this case, ``len(x) == len(y) + 1``
+            stepMode     (str or None) If "center", a step is drawn using the x
+                         values as boundaries and the given y values are
+                         associated to the mid-points between the boundaries of
+                         each step. This is commonly used when drawing
+                         histograms. Note that in this case, len(x) == len(y) + 1
+                         If "left" or "right", the step is drawn assuming that
+                         the y value is associated to the left or right boundary,
+                         respectively. In this case len(x) == len(y)
+                         If not passed or an empty string or None is passed, the
+                         step mode is not enabled.
+                         Passing True is a deprecated equivalent to "center".
                          (added in version 0.9.9)
+                         (str modes added in version 0.12.0)
```

Comment:
I would go ahead and remove the `modes` added in version 0.12.0; we'll likely have this in 0.11.1....

Reply:
73542aa does it